RealmModelGenerator/AttributeDetailView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AttributeDetailView.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/29/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+@objc protocol AttributeDetailViewDelegate {
```

Comment:
we are going to drop the objc prefix and just have the protocol conform to class
also all of the methods will be non optional because swift doesn't support optional protocols yet extensively
we need to drop obj because we are not allowed to use optionals for method parameters.